.storybook/main.js

```diff
@@ -76,16 +64,18 @@ module.exports = {
     }
     return config;
   },
-  managerWebpack: async (config) => {
+  managerWebpack: async config => {
     // Build speed improvements
     applySpeedTweaks(config);
-
     if (process.env.MEASURE) {
       const smp = new SpeedMeasurePlugin();
       return smp.wrap(config);
     }
     return config;
   },
+  docs: {
+    autodocs: true
+  }
```

Comment:
I like the autodocs! They might need some work to get fully functional, though.  `KoboPrompt` and `KoboModal` don't close and obscure the docs. `KoboDropdown` and `KoboSelect` also clip outside of the doc view's bounds when expanded, but the docs are still accessible and the story displays fine on the Primary tab so that's probably not worth spending much energy on.

Reply:
A migration added this. I could go either way on it. Could disable it or just ignore the defects for now.

Reply:
I went back and forth about it, and I think the immediate benefit for frontend devs from having code examples outweighs having some wonky Storybook bits, so I'm approving. If you don't mind, could you create an issue/task to update the docs pages?